expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -277,6 +253,41 @@ void shouldNotFailWhenInputContainsAffiliationsThatAreIncomplete() {
         assertDoesNotThrow(() -> ExpandedResource.fromPublication(uriRetriever, publication));
     }
 
+
+    private void assertExplicitFieldsFromFraming(ObjectNode framedResultNode) {
+        var node = framedResultNode.at("/topLevelOrganization");
+        StreamSupport.stream(node.spliterator(), false)
+            .flatMap(ExpandedResourceTest::getaVoid)
+            .forEach(ExpandedResourceTest::assertAllFieldsAreValid);
+    }
+
+    private static Stream<String> getaVoid(JsonNode topOrg) {
+        var spliterator = Spliterators.spliteratorUnknownSize(topOrg.fieldNames(), Spliterator.ORDERED);
+        return StreamSupport.stream(spliterator, false);
+    }
+
+    private static void assertAllFieldsAreValid(String fieldName) {
```

Comment:
The method name does not really describe what the method does. There is no "all" inside the method. Maybe rename to assertFieldIsValid instead?